.github/workflows/terraform-checks.yaml

```
@@ -17,10 +17,16 @@ on:
         required: false
         type: string
         default: latest/stable
+      apply:
+        description: Boolean value that defines if `apply` job runs. Defaults to `True`.
+        required: false
+        type: boolean
+        default: True
```

Having the boolean in lowercase seems more fitting. Also don't forget to change the description
```suggestion
        default: true
```

---

Thanks, updated!